raiden/network/transport/matrix/transport.py

```diff
@@ -1267,80 +1259,98 @@ def _leave_unused_rooms(self, _address_to_room_ids: Dict[AddressHex, List[_RoomI
         )
         return
 
-    def send_to_device(self, address: Address, message: Message) -> None:
-        """ Sends send-to-device events to a all known devices of a peer without retries. """
-        user_ids = self._address_mgr.get_userids_for_address(address)
-
-        data = {user_id: {"*": MessageSerializer.serialize(message)} for user_id in user_ids}
-
-        return self._client.api.send_to_device("m.to_device_message", data)
+    def reachability_announcement(self, reachability: AddressReachability) -> None:
+        if self._raiden_service:
+            while not self._stop_event.ready():
+                self.log.debug("Announcing reachability to all known peers")
+                message = ReachabilityNotification(
+                    address_reachability=reachability.value, signature=EMPTY_SIGNATURE
+                )
+                self._raiden_service.sign(message)
```

Comment:
I have some strong concerns about this. Requiring a new signature to be made every 10s shouldn't be made without syncing with the Light Client team. This actually is very unfeasible for us, as all signatures are interactive (for now), and even after we implement subkey for non-interactive signing, the former case will still be supported and would be worse than it's already is with this change.